src/index.ts

```diff
@@ -75,6 +76,10 @@ function onDelayTouchstart( evt:TouchEvent ) {
         clearTimeout( timer );
```

Comment:
Whats your opinion on adding another event here in case the dragstart is canceled?

E.g.
```TS
if (el) {
    el.dispatchEvent(new CustomEvent(EVENT_DRAG_CANCELED, { bubbles: true, cancelable: true }))
}
```
where `EVENT_DRAG_CANCELED` is named `dnd-poly-dragstart-cancel`

See my other comments for suggestions regarding the event name.

Reply:
Added dnd-poly-dragstart-cancel, Please review

Reply:
Can you move the code for dispatching the cancel event at line 75/76? 

In the `dragOperationEnded` at line 103 where it's at the moment it will not fire when the drag will not start because of cancel reasons related to `holdToDrag`. I'd like to keep the events you added exclusively related to the `holdToDrag` feature.

I tried it out in the demo and it works very well! Great job :)

Reply:
Thanks, I move the code inside onReleasedItem.